Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -24,6 +24,95 @@ Function CmdletHasMember($memberName) {
     return $cmdletParameter
 }
 
+#To parse the Overriding Parameters string provided by user
+Function GetPropertiesMapping {
+    param ( [String] $overridingParameters )
```

Comment:
Start with Capitals

Reply:
used PascalCase

---
In reply to: [137521169](https://github.com/Microsoft/vsts-tasks/pull/5246#discussion_r137521169) [](ancestors = 137521169)